test/e2e/README.md

```diff
@@ -0,0 +1,25 @@
+# End-to-End testing
+
+## Running the tests
+
+To run all e2e tests:
+```
+GOCACHE=off go test -tags e2e ./test/e2e/...
```

Comment:
This info seems to conflict with using `count=1` as per https://golang.org/doc/go1.10#test